src/wallet.ts

```diff
@@ -1087,9 +1090,13 @@ export class Wallet {
     const obj = {
       owner_id: accountId,
       metadata: {
+        title: metadata?.title,
+        description: metadata?.description,
+        media: metadata?.media,
+        media_hash: metadata?.media_hash,
```

Comment:
Shall we keep these here?

@tifrel @cif

Reply:
Why not. But we need to be sure that if media is set, media_hash is set as well

Reply:
Yes, when uploading through MintbaseJS `media_hash` is always generated.